ci/docker/runtime_functions.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
```

Comment:
Move file to QEMU folder since it's only related to QEMU?

Reply:
ok